common/system.h

```diff
@@ -120,6 +120,25 @@ enum Type {
 
 } // End of namespace LogMessageType
 
+/**
+* Pixel mask modes for cursor graphics.
+*/
+enum CursorMaskValue {
+	// Overlapped pixel is unchanged
```

Comment:
Please mark these as doxygen comments, e.g. similar to `kCursorMaskInvert` below